ComputeClient/Compute.Contracts/Requests/Snapshot/SnapshotServicePlanListOptions.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DD.CBU.Compute.Api.Contracts.Requests.Snapshot
+{
+	using System;
+
+	/// <summary>
+	/// Filtering options for the Snapshot Server Plan List request.
+	/// </summary>
+	public sealed class SnapshotServicePlanListOptions : FilterableRequest
+	{
+		/// <summary>
+		/// The "id" field name.
+		/// </summary>
+		public const string IdField = "id";
+
+		/// <summary>
+		/// The "available" field name.
+		/// </summary>
+		public const string availableField = "available";
```

Comment:
Is the convention for field names to be uppercase or lowercase for first letter?

Reply:
+1 good spot - yeah by convention it should be upper case will fix